From: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Bugs-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=613411#43
Description: Explicitly link against libraptor2
 The lib is indirectly included via librdf0, but without linking to
 libraptor2, we won't get the versioned symbols referenced in libslv2.
Forwarded: should we?
--- a/wscript
+++ b/wscript
@@ -94,6 +94,7 @@
 	obj.includes     = ['.', './src']
 	obj.name         = 'libslv2'
 	obj.target       = 'slv2'
+	obj.lib          = 'raptor2'
 	obj.vnum         = SLV2_LIB_VERSION
 	obj.install_path = '${LIBDIR}'
 	autowaf.use_lib(bld, obj, 'REDLAND LV2CORE')
